src/App.tsx

```diff
@@ -15,48 +16,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  alphabetically = 'alphabet',
+  byLength = 'length',
```

Comment:
prefix by is redundant 
```suggestion
  Length = 'length',
```